.c3i/config_v2.yml

```diff
@@ -46,9 +46,9 @@ tasks:
     update_labels: false
     user_feedback:
       title: "Conan v2 pipeline"
-      description: "> **Note**: Conan v2 builds may be required once they are on the [v2 ready](https://github.com/conan-io/conan-center-index/blob/master/.c3i/conan_v2_ready_references.yml) list"
+      description: "> **Note**: Conan v2 builds are now mandatory. Please read our [discussion](https://github.com/conan-io/conan-center-index/discussions/19104) about it."
       regression: "> **Regression**: Conan v2 builds are mandatory and they are required for the PR to be merged, because this recipe worked with Conan v2 previously."
-      text_on_failure: "The v2 pipeline failed. Please, review the errors and note this will be required for pull requests to be merged in the near future."
+      text_on_failure: "The v2 pipeline failed. Please, review the errors and note this is required for pull requests to be merged. In case this recipe is still not ported to Conan 2.x, please, comment on the PR and we will help you."
```

Comment:
maybe we need a more specific message to ask for help. Maybe a gh mention with "@___" do we have a "@maintainers" group?

Reply:
Yes, so far we have Barbarians group, but it alerts everyone. We could provide another group. As groups are created by organization, @maintainers would be too generic, what about @cci-maintainers then?

Reply:
let's start with @conan-io/barbarians

Reply:
Done, included conan/barbarians

Reply:
You wont get notifications from non-members of the group AFAIK on GitHub

Reply:
@prince-chrismc I didn't know about that. Checking the documentation, it seems we can configure notifications: https://docs.github.com/en/organizations/organizing-members-into-teams/configuring-team-notifications

Reply:
Ohhh perfect! Glad they changed that